alerts/state.libsonnet

```diff
@@ -36,6 +36,34 @@
               severity_level: 'warning',
             },
           },
+          {
+            alert: 'CephOSDVersionMismatch',
+            expr: |||
+              count(count(ceph_osd_metadata{%(cephExporterSelector)s}) by (ceph_version)) > 1
+            ||| % $._config,
+            'for': '1h',
+            labels: {
+              'for': $._config.clusterVersionAlertTime,
+            },
+            annotations: {
```

Comment:
Currently we are following the following format for the annotations : 
`
annotations: {
              message: '',
              description: '',
              storage_type: '',
              severity_level: '',
            }
`
Please add the missing fields.

Reply:
created issue for this https://github.com/ceph/ceph-mixins/issues/54